packages/edit-post/src/components/layout/index.js

```diff
@@ -265,8 +266,8 @@ function Layout( { styles } ) {
 				}
 				footer={
 					! isDistractionFree &&
-					showBlockBreadcrumbs &&
 					! isMobileViewport &&
+					showBlockBreadcrumbs &&
```

Comment:
Nothing appears to have changed here, but the `isDistractionFree` variable no longer contains the viewport width condition.

This fixes another problem. In trunk, there is a "gap" in the viewport width generated by this conditional statement, so the breadcrumb list is displayed when the viewport width is between 782px and 960px, even in distraction-free mode, as shown below.

https://user-images.githubusercontent.com/54422211/223905311-f0ed5a15-d403-4de1-9501-0a0503b7e862.mp4